api/src/pcapi/routes/backoffice/templates/components/history/actions.html

```diff
@@ -32,6 +32,13 @@
               </p>
             {% endif %}
             {% if action.extraData %}
+              {% set role_name = action.extraData.get('role_name', '') %}
+              {% if role_name %}
+                <div>
+                  <span class="fw-bold">Nom du rôle :</span>
+                  <div>{{ role_name }}</div>
```

Comment:
Suggestion : pour éviter de prendre trop de place à l'écran, ça devrait tenir sur une seule ligne : 
```suggestion
                  <span class="fw-bold">Rôle :</span>
                  {{ role_name }}
```